query/common/src/main/java/org/codice/ddf/admin/common/fields/base/BaseObjectField.java

```diff
@@ -40,9 +41,13 @@ public BaseObjectField(String fieldName, String fieldTypeName, String descriptio
     public Map<String, Object> getValue() {
         Map<String, Object> values = new HashMap<>();
 
-        for(Field field : getFields()) {
-            if(!(field instanceof FunctionField)) {
-                values.put(field.fieldName(), field.getValue());
+        for (Field field : getFields()) {
+            if (!(field instanceof FunctionField)) {
+                if (field.fieldName().equals(PasswordField.DEFAULT_FIELD_NAME) && ((PasswordField) field).isInternalProcess()){
```

Comment:
This is supposed to identify if the caller is a test or the graphQL logger. Tests are currently using this getValue method to create arguments and they need the actual password.